src/core.rs

```diff
@@ -316,6 +591,11 @@ impl Revision {
             .host(&target.host)
             .profile("Debug");
 
+        for (key, value) in build_config.as_cmake_define_key_values() {
+            println!("setting {}={}", key, value);
```

Comment:
Is this debugging code?

Reply:
Ah, yes - at first I wanted to make sure that the user could see their settings take flight - but then I noticed it's passed in the call to cmake anyway. I'll remove this in a follow up. Thanks for the fast review and merge 🙇 .